test/end-to-end/pages/payment/govukconfirmpayment.js

```diff
@@ -1,11 +1,15 @@
 'use strict';
+const contentEn = 'Payment summary';
+const contentCy = 'Crynodeb o\'r taliad';
 
-module.exports = async function() {
+module.exports = async function(language ='en') {
     const I = this;
+    const commonContent = language === 'en' ? contentEn : contentCy;
 
-    await I.waitForText('Payment summary');
+    if (language === 'en') {
+        await I.waitForText(commonContent);
+    }
```

Comment:
again, not sure why you'd only want to check the text here on English journeys? It will likely help with test stability for both languages